wp_api_integration_tests/tests/test_tags_immut.rs

```diff
@@ -90,6 +120,22 @@ mod filter {
             });
     }
 
+    #[apply(sparse_tag_field_with_edit_context_test_cases)]
+    #[case(&[SparseTagFieldWithEditContext::Name, SparseTagFieldWithEditContext::Slug])]
+    #[tokio::test]
+    #[parallel]
+    async fn filter_retrieve_posts_with_edit_context(
```

Comment:
The filter retrieve test names refer to `posts`. I am fixing these in the `/categories` PR I am working on right now to avoid merge conflicts.